pkgs/by-name/li/libdnet/package.nix

```diff
@@ -28,7 +28,10 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
     check
```

Comment:
You pass check also here, could you remove it?

Reply:
I'll check if it can be removed